EvidenceApi/V1/Filters/HandleExceptionAttribute.cs

```diff
@@ -0,0 +1,47 @@
+using JetBrains.Annotations;
```

Comment:
This file has been in added in a `FIlters` directory at the top level but we could just as easily nest it in another directory, e.g. `Controllers/Filters`.

Reply:
For better visibility I think it should be in the `Controllers/Filters`